.vscode/settings.json

```
@@ -62,10 +61,9 @@
     "pwsh": null
   },
```

```suggestion
  },
  
  "terminal.integrated.profiles.windows": {
  },
  
```

🔗 https://code.visualstudio.com/docs/terminal/profiles#_git-bash